BITSManager/SetJobPropertyControl.xaml.cs

```diff
@@ -52,6 +54,15 @@ public void SetJobProperties(BITS.IBackgroundCopyJob job)
                 }
             }
 
+            if (jobHttpOptions != null)
+            {
+                var text = _uiCustomHeadersAll.Text;
+                if (!String.IsNullOrWhiteSpace (text))
```

Comment:
> ( [](start = 46, length = 2)

Nit: there shouldn't be any whitespace before the parenthesis. Same in line 62

Reply:
I asked Visual Studio to reformat the entire file, and the parenthesis now follow the correct standard. This is the only human-edited .CS file in the PR, so there aren't any other files to fix up.

---
In reply to: [248035881](https://github.com/Microsoft/BITS-Manager/pull/11#discussion_r248035881) [](ancestors = 248035881)